app/helpers/posts_helper.rb

```diff
@@ -1,2 +1,23 @@
 module PostsHelper
+  def render_text_section(post)
+    content_tag(:div, class: 'text') do
+      concat content_tag(:p, "Comments: #{post.CommentsCounter.nil? ? 0 : post.CommentsCounter}",
+                         class: 'comment-text')
+      concat content_tag(:p, "Likes: #{post.LikesCounter.nil? ? 0 : post.LikesCounter}")
+    end
+  end
+
+  def render_comments_section(post)
+    content_tag(:ul, class: 'ul-comments-box') do
+      concat content_tag(:h4, 'Section for Comments:', class: 'section-comments')
+
+      if post.comments.blank?
+        concat content_tag(:li, 'no comments for the moment')
+      else
+        post.comments.each do |comment|
+          concat content_tag(:li, "Username: #{comment.Text.blank? ? 'no comments for the moment' : comment.Text}")
+        end
+      end
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=page%20to%20another-,Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.,-Hint%3A%20In), You used this method to render comments in post index view, please use the `recent_comments` method inside post model to render the most recent 5 comments inside `post index view`.